pubspec.yaml

```diff
@@ -1,16 +1,16 @@
 name: graphs
-version: 0.2.1-dev
+version: 0.3.0-nullsafety.0
 description: Graph algorithms that operate on graphs in any representation
 homepage: https://github.com/dart-lang/graphs
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
-
+  sdk: '>=2.12.0-0 <3.0.0'
+dependency_overrides:
```

Comment:
drop this. move back to a normal dev_dependency at `^1.0.0`

Reply:
done